docs/static/pipeline-viewer.asciidoc

```diff
@@ -46,15 +56,17 @@ image::static/docs/images/monitoring-ui.png[Monitoring UI]
 being monitored.
 +
 Each pipeline is identified by a pipeline ID (`main` by default). For each
-pipeline, you'll see a list of all versions of the pipeline running during
-the specified time range.
+pipeline, you'll see a list of all versions of the pipeline stats that were
```

Comment:
In addition to the statement you requested, I added a bit more text here so users don't think they're viewing a configuration that's been saved somewhere.